internal/pkg/constants/constants.go

```diff
@@ -8,6 +8,8 @@ var (
 )
 
 const (
+	RootCASecretName          = "kubewarden-root-ca"
+	ControllerCertsSecretname = "kubewarden-controller-certificate"
```

Comment:
Silly nitpick with CamelCase:

```suggestion
	ControllerCertsSecretName = "kubewarden-controller-certificate"
```